pkgs/data/icons/quintom-cursor-theme/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenvNoCC, fetchFromGitLab, lib }:
+  stdenvNoCC.mkDerivation {
```

Comment:
Is there a new version of `nixpkgs-fmt` that prepends 2 spaces before a function call?
If it's not that, I'm not sure if this is a good idea. I haven't come across derivations where those spaces are present.

Reply:
I have never heard of this tool before, thanks for bringing it to my attention since it wasn't mentioned in any of the docs about PRs or contributing to nixpkgs that I read. Formatting fixed.